Commander X16 Programmer's Reference Guide.md

```diff
@@ -1832,13 +1847,19 @@ The Commander X16 has 64 KB of ROM and 2,088 KB (2 MB[^1] + 40 KB) of RAM. Some
 
 This is an overview of the X16 memory map:
 
-|Addresses  |Description                                                       |
-|-----------|------------------------------------------------------------------|
-|$0000-$9EFF|Fixed RAM (40 KB minus 256 bytes)								   |
-|$9F00-$9FFF|I/O Area (256 bytes)											   |
-|$A000-$BFFF|Banked RAM (8 KB window into one of 256 banks for a total of 2 MB)|
+|Addresses  |Description                                                          |
+|-----------|---------------------------------------------------------------------|
+|$0000-$0399|System Reserved (see RAM Contents section)                           |
```

Comment:
This is an overview form a hardware perspective. The special casing of 0000-03FF only applies from a software perspective.